### ISSUE TITLE
Using git's pager by any means breaks colored output.

### ISSUE THREAD

**Issue by mschnee** [Jaykul/PowerLine#15]
git commands that invoke the pager (for paged output, e.g. `fetch --all`, `diff`, etc) have the side effect of breaking the color outputting of the prompt.

![image](https://cloud.githubusercontent.com/assets/1375316/25297493/f7cbddc0-26a2-11e7-9992-c20e0849bedb.png)

This also effects all embedded terminales (Visual Studio, Visual Studio Code)
![image](https://cloud.githubusercontent.com/assets/1375316/25297528/388a9a22-26a3-11e7-91d1-7fdc54c3cd1e.png)

This does not affect ConEmu:
![image](https://cloud.githubusercontent.com/assets/1375316/25297507/1404faf8-26a3-11e7-962f-fff2b30ee8fd.png)

## Configuration Used
```ps
#requires -module PowerLine
using module PowerLine

$PowerLinePrompt = @(
        @{ bg = "Cyan";     fg = "White"; text = { $MyInvocation.HistoryId } }
        @{ bg = "DarkBlue"; fg = "White"; text = { $pwd } }
    )

Set-PowerLinePrompt -PowerLineFont
```

## OS
```
Edition      Windows 10 Pro
Version      1703
OS Build     15063.138
```

This is very strange, especially since the Windows 10 Creator's Update added a [ton of support](https://blogs.msdn.microsoft.com/commandline/2017/04/11/windows-10-creators-update-whats-new-in-bashwsl-windows-console/) and functionality to the console.

There's a possiblity that git's msys less.exe is outputting something bad that is then breaking all further output?

**Comment by michaeltlombardi** [comment]
@Jaykul, did you mention at the summit that this is fixed in master but not in release?

**Comment by ayrtonmassey** [comment]
I'm also seeing this issue when using git commands e.g. `git push` or building a project via webpack. Some commands fix the broken prompt, e.g. `code .`.

## OS

- Edition: `Windows 10 Enterprise`
- Version: `1607`
- OS Build: `14393.1198`

## PowerShell

- PSVersion: `5.1.14393.1198`

## PowerLine Settings

- Version: `2.0.0`
- Config:
```posh
using module PowerLine

$PowerLinePrompt = @(
        @{ bg = "Cyan";     fg = "White"; text = { $MyInvocation.HistoryId } }
        @{ bg = "DarkBlue"; fg = "White"; text = { $pwd } }
    )

Set-PowerLinePrompt -PowerLineFont
```

**Comment by Xainey** [comment]
I noticed this issue as well when running a few common tasks with git and npm/webpack using hyper.is. I will keep an eye out for this to happen again and try to capture the cause.

**Comment by Jaykul** [comment]
This should be fixed in the recently released 2.3 -- but warning: that release doesn't work in ISE.
